packages/cli/api-importers/openapi/openapi-ir-parser/src/schema/convertSchemas.ts

```diff
@@ -522,6 +522,20 @@ export function convertSchemaObject(
         });
     }
 
+    // null type (OpenAPI 3.1 spec)
+    if ((schema.type as string) === "null") {
+        return SchemaWithExample.unknown({
+            nameOverride,
+            generatedName,
+            title,
+            description,
+            availability,
+            namespace,
+            groupName,
+            example: null
+        });
+    }
```

Comment:
can you add a fixture in `openapi-ir-to-fern-tests` ? so we can have a test that covers this code path?